pkg/apis/performance/v1alpha1/performanceprofile_types.go

```diff
@@ -12,7 +12,15 @@ type PerformanceProfileSpec struct {
 	CPU *CPU `json:"cpu,omitempty"`
 	// HugePages defines set of huge pages related parameters.
 	HugePages *HugePages `json:"hugepages,omitempty"`
-	// NodeSelector is a selector which must be true for the performance profile to fit on a node.
+	// MachineConfigLabels defines the labels to add to the MachineConfigs the operator creates. They have to be
+	// used as MachineConfigSelector in the MachineConfigPool which targets this performance profile.
+	MachineConfigLabels map[string]string `json:"machineConfigLabels,omitempty"`
+	// MachineConfigPoolSelector defines the MachineConfigPool labels to use in the MachineConfigPoolSelector
+	// of resources like KubeletConfigs created by the operator.
+	MachineConfigPoolSelector map[string]string `json:"machineConfigPoolSelector,omitempty"`
+	// NodeSelector defines the Node labels to use in the NodeSelectors of resources like Tuned created by the operator.
+	// They most likely should, but do not need to match the node labels in the NodeSelector of the MachineConfigPool
+	// which targets this performance profile.
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
```

Comment:
Can you rename it to `TunedNodeSelector`

Reply:
not sure... we might need node selectors for other feature in future as well? The comment states that this is used for Tuned atm.